### DISCUSSION TITLE
Simple IP-ACL?

### DISCUSSION THREAD

**Discussion by barhom** [akvorado/akvorado#1490] [Q&A]
Hello,

Is there any easy way to add an easy IP-ACL without any authentication?
Tried changing a lot of things in the docker-compose traefik section to no avail.

**Comment by vincentbernat** [comment]
There is https://doc.traefik.io/traefik/middlewares/http/ipallowlist/ that should work.

**Comment by kronicklez** [comment]
Hi @vincentbernat ,

May i know where is located file "docker-compose traefik" so i can learn to change it?

Thanks

**Comment by barhom** [comment]
@vincentbernat Ive used ip-acl according to your example many times but quite unsure exactly on what router instance I need to implement it to akvorados docker.

Would you maybe include it as an example in the project or documentation? I think it's quite useful.

**Reply by vincentbernat** [reply]
For console:
```
    labels:
      - traefik.enable=true
      - traefik.http.routers.akvorado-console.rule=PathPrefix(`/`)
      - traefik.http.routers.akvorado-console.middlewares=console-auth-headers,console-ipallowlist
      - traefik.http.middlewares.console-ipallowlist.ipallowlist.sourcerange=192.168.0.0/16
      - traefik.http.middlewares.console-auth-headers.headers.customrequestheaders.Remote-User=alfred
      - traefik.http.middlewares.console-auth-headers.headers.customrequestheaders.Remote-Name=Alfred Pennyworth
      - traefik.http.middlewares.console-auth-headers.headers.customrequestheaders.Remote-Email=alfred@example.com
      - traefik.http.services.akvorado-console.loadbalancer.server.port=8080
```
But it could be applied to other components as they also expose services. Maybe this is possible directly in traefik service.

**Reply by kronicklez** [reply]
Hi @vincentbernat ,

The config "labels" not just copy paste in console.yaml right? I just want try change name "alfred" to other name.

Thanks and appreciate your feedback

**Comment by barhom** [comment]
Didn't really work. I guess I'll see if I can run another traefik infront of the built in traefik on port 80/443 and forward to it.
This is mainly so that I do not change too much of the akvorado config.
